.github/workflows/pr_iac_lint.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         with:
           distribution: 'corretto'
           java-version: 11
+      - name: Build Project
+        working-directory: .
+        run: |
+          mvn install -DskipTests
```

Comment:
`terraform validate` needs to be able to find the artifacts of the build process. Because they are all `-SNAPSHOT`, they have to be built. So, build them.